scripts/flask/web_interface.py

```
@@ -18,6 +19,29 @@
+
+    if len(arg_list) % 2 != 0:
+        return False, "Arguments must be in flag/value pairs"
+
+    for i in range(0, len(arg_list), 2):
+        flag = arg_list[i]
+        value = arg_list[i + 1]
+
+        if flag not in allowed_flags:
+            return False, f"Unexpected flag: {flag}"
+
+        arg_def = allowed_flags[flag]
```

Ruff (0.11.9)

Local variable `arg_def` is assigned to but never used

Remove assignment to unused variable `arg_def`

(F841)